### PR TITLE
[SRE] clean up & housekeeping

### PR DESCRIPTION
**Pull request by geisslet**
- replace the useless sophisticated by simple availability (last hour) alerts to trigger action
- this lead to changes in the aggregation and a shift to percentage SLO values

### CODE REVIEW

File: openstack/sre/templates/prometheus-alerts.yaml [hunk 1]
```diff
@@ -13,48 +13,24 @@ spec:
...
-            (100 * global:api_errors_per_request_sli:ratio_rate6h > on (ingress) global:api_errors_per_request_slo:percent)
-          )
-      for: 1h
+    - alert: ApiAvailabilityCritical
+      expr: global:api_errors_per_request_sli:ratio_rate1h <= on (ingress,region) global:api_availability_critical_slo:percent
       labels:
         context: availability
         service: "{{`{{$labels.ingress}}`}}"
-        severity: info
+        severity: warning
```
Comment (joker-official) [inline openstack/sre/templates/prometheus-alerts.yaml:32]: Why just warning here? It's supposed to be critical in the end, right?

File: openstack/sre/templates/prometheus-alerts.yaml [hunk 2]
```diff
@@ -13,48 +13,24 @@ spec:
...
-          )
-          or
-          (
-            (100 * global:api_errors_per_request_sli:ratio_rate3d > on (ingress) global:api_errors_per_request_slo:percent)
-            and
-            (100 * global:api_errors_per_request_sli:ratio_rate6h > on (ingress) global:api_errors_per_request_slo:percent)
-          )
-      for: 1h
+    - alert: ApiAvailabilityCritical
+      expr: global:api_errors_per_request_sli:ratio_rate1h <= on (ingress,region) global:api_availability_critical_slo:percent
```
Comment (joker-official) [inline openstack/sre/templates/prometheus-alerts.yaml:28]: Don't we need the same thing as above here, too? I mean the `(1 - X) * 100`?

File: openstack/sre/templates/prometheus-aggregations.yaml [hunk 3]
```diff
@@ -14,13 +14,20 @@ spec:
...
     rules:
-{{- range $api, $slo := $values.slo.api_error_ratio_percent }}
-    - record: global:api_errors_per_request_slo:percent
+{{- range $api, $slo := $values.slo.api_availability_percent }}
+    - record: global:api_availability_warning_slo:percent
       labels:
           ingress: {{ $api | quote }}
           region: {{required ".Values.global.region missing" $values.global.region}}
-      expr: {{ $slo | quote }}
+      expr: {{ $slo.warning | default $values.slo.default_warning  | quote }}
```
Comment (joker-official) [inline openstack/sre/templates/prometheus-aggregations.yaml:22]: One space too much in front of `| quote`.

File: openstack/sre/values.yaml [hunk 4]
```diff
@@ -9,40 +9,42 @@ prometheus:
...
-    maia: 0.1
-    maia-oprom: 0.1
-    manila-api: 0.1
-    neutron-server: 0.1
-    nova-api: 0.1
-    nova-console: 0.1
-    nova-placement-api: 0.1
-    octavia-api: 0.1
+  default_warning: 99.8
+  default_critical: 99.6
```
Comment (joker-official) [inline openstack/sre/values.yaml:20]: Imho, putting the defaults here just as "default_warning" doesn't make sense if we ever want to support more than one metric, which having "api_availability_percent" as an extra dict implies. It needs to somehow align with supporting that, e.g.
```
slo:
  defaults:
    api_availability_percent:
      warning: 99.8
      critical: 99.6
```

That being said, it currently doesn't matter and could be done once we need more than one default or metric. Your choice.

File: openstack/sre/templates/prometheus-alerts.yaml (hunk 1 above)
Comment (geisslet) [reply openstack/sre/templates/prometheus-alerts.yaml:32]: true

File: openstack/sre/templates/prometheus-alerts.yaml (hunk 2 above)
Comment (geisslet) [reply openstack/sre/templates/prometheus-alerts.yaml:28]: true // sorry

File: openstack/sre/templates/prometheus-aggregations.yaml (hunk 3 above)
Comment (geisslet) [reply openstack/sre/templates/prometheus-aggregations.yaml:22]: ok

File: openstack/sre/values.yaml (hunk 4 above)
Comment (geisslet) [reply openstack/sre/values.yaml:20]: very true

**Pull request merged**